Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +178,20 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDistance() {
+        return convertToInches(leftDrive.getPosition() - startPosition.get());
```

Comment:
Are you saying that there was a technical problem preventing you from getting data? What was the behavior?

You bring up a good point; it would be nice to be able to operate on just a single encoder if something gets disconnected.